File: src-tauri/src/entity/account.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Serialize, Deserialize)]
pub(crate) struct Account {
    pub(crate) id: Option<i32>,
    pub(crate) name: String,
    pub(crate) username: String,
    pub(crate) password: String,
    pub(crate) sequence: Option<i32>,
    pub(crate) liked: Option<bool>,
    pub(crate) description: Option<String>,
    pub(crate) last_update_time: Option<String>,
    // 外部字段
    pub(crate) account_category_ids: Option<Vec<i32>>,
}

File: src/account.ts
export { Account };

class Account {
	public id: number | null = null;
	public name: string = '';
	public username: string = '';
	public password: string = '';
	public sequence: number = 1;
	public liked: boolean = false;
	public description: string | null = null;
	public last_update_time: string | null = null;
	public show: boolean = false;
	public account_category_ids: number[] = [];

	constructor() {}
}
